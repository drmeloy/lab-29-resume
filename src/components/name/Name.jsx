import React from 'react';
import PropTypes from 'prop-types';

const Name = ({ name }) => {
  return(
    <h1>{name}</h1>
  );
};

Name.propTypes = {
  name: PropTypes.string.isRequired
}

export default Name;
