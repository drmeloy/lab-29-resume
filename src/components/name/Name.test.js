import React from 'react';
import { shallow } from 'enzyme';
import Name from './Name.jsx';

describe('Name component', () => {
  it('renders a Name component', () => {
    const wrapper = shallow(<Name name={''} />);
    expect(wrapper).toMatchSnapshot();
  })
});
