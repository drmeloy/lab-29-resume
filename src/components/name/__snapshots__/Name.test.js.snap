// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name component renders a Name component 1`] = `<h1 />`;
